pkg/addons/data/ServiceAccount/openshift-infra/cluster-reader.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cluster-reader
```

Comment:
how do you feel about being more specific here?  This is for Microsoft SRE, so maybe microsoft-sre?  I'm also wondering if we want to separate anything we do specifically for OSA into a different namespace.  Didn't we just get rid of the openshift-azure one? :rofl:  

What I'm wondering here is if we need to be wary of things in the future that make assumptions on what's in the infra namespace and remove or change things we've added.

Reply:
> Didn't we just get rid of the openshift-azure one? rofl 

Was thinking about this too. My vote would be `openshift-azure` namespace and dont distinguish our ops and their ops. Just `azure-cluster-reader`. We can create a dedicated role and role binding with `azure` prefix (or postfix). And so we know that anything with prefix `azure` is our made.

Reply:
+1 on using openshift-azure for OSA-specific things.